### ISSUE TITLE
Automatic thumbnail creation with nautilus is missing

### ISSUE THREAD

**Issue by belegdol** [dbrgn/miniaturo#13]
In order for miniaturo to be a drop-in replacement for raw-thumbnailer, it also needs to create the thumbnails automatically. I believe issue #11 is supposed to be a proper solution. In the meantime, I adapted the files shipped with raw-thumbnailer so that the thumbnails are at least created for the mime types supported by raw-thumbnailer so that no functionality is lost by switching. I do not know how to have them installed hence no PR.
[miniaturo.zip](https://github.com/dbrgn/miniaturo/files/11365177/miniaturo.zip)

**Comment by dbrgn** [comment]
I'm still trying to find out, when and how raw-thumbnailer / miniaturo is being used / integrated 🙂

I'm personally using Thunar, not Nautilus. Thunar uses [tumbler](https://docs.xfce.org/xfce/thunar/4.14/tumbler), which is a dbus service to automatically generate thumbnails. Nowadays, it includes a built-in [raw-thumbnailer plugin](https://gitlab.xfce.org/xfce/tumbler/-/tree/master/plugins/raw-thumbnailer?ref_type=heads), so conversion of RAW files works out of the box. (This wasn't yet the case when I wrote miniaturo initially.)

They also added support for custom `.thumbnailer` configs, but if it's built-in, why add a separate binary that uses the same library under the hood?

Regarding Nautilus: Does someone know what the plans are? Will a `.thumbnailer` file based config be used for the foresesable future? Will they also rely on DBUS (see #11)?

In other words, is miniaturo still needed, or is it wasted effort?

(CC @hfiguiere, maybe you know more?)

**Comment by hfiguiere** [comment]
all I know it's not packaged in Fedora.
